adverse_action.go

```diff
@@ -124,7 +124,7 @@ func (c *Client) GetAdverseAction(adverseActionID string) (*AdverseAction, error
 	}
 
 	if resp.StatusCode != http.StatusOK {
-		return nil, NewError([]int{
+		return nil, NewResponseError([]int{
 			http.StatusCreated,
```

Comment:
@chandradeepak it seems like I made a mistake here before, can you please change this to `http.StatusOK`?